Schweizerisches Bundesgericht.js

```diff
@@ -241,3 +241,169 @@ function doWeb(doc, url) {
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/de/php/aza/http/index.php?lang=de&type=simple_query&query_words=Unterhalt&lang=de&top_subcollection_aza=all&from_date=&to_date=",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/fr/php/clir/http/index.php?lang=fr&type=highlight_simple_query&page=1&from_date=&to_date=&from_year=1954&to_year=2021&sort=relevance&insertion_date=&from_date_push=&top_subcollection_clir=bge&query_words=UNterhalt&part=all&de_fr=&de_it=&fr_de=&fr_it=&it_de=&it_fr=&orig=&translation=&rank=1&highlight_docid=atf%3A%2F%2F81-I-63%3Ade&number_of_ranks=714&azaclir=clir",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": "ATF 81 I 63",
+				"creators": [],
+				"dateDecided": 1955,
+				"abstractNote": "Militärpflichtersatz: Besteuerung eines Evangelisten, für dessen Unterhalt seine Glaubensfreunde sorgen.",
+				"court": "Tribunal Fédéral Suisse",
+				"docketNumber": "ATF 81 I 63",
+				"url": "https://www.bger.ch/ext/eurospider/live/fr/php/clir/http/index.php?lang=fr&type=highlight_simple_query&page=1&from_date=&to_date=&from_year=1954&to_year=2021&sort=relevance&insertion_date=&from_date_push=&top_subcollection_clir=bge&query_words=UNterhalt&part=all&de_fr=&de_it=&fr_de=&fr_it=&it_de=&it_fr=&orig=&translation=&rank=1&highlight_docid=atf%3A%2F%2F81-I-63%3Ade&number_of_ranks=714&azaclir=clir",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/fr/php/aza/http/index.php?highlight_docid=aza%3A%2F%2Faza://27-10-2021-8C_329-2021&lang=fr&zoom=&type=show_document",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": "TF 8C_329/2021",
+				"creators": [],
+				"dateDecided": "27.10.2021",
+				"court": "Tribunal Fédéral Suisse",
+				"docketNumber": "8C_329/2021",
+				"url": "https://www.bger.ch/ext/eurospider/live/fr/php/aza/http/index.php?highlight_docid=aza%3A%2F%2Faza://27-10-2021-8C_329-2021&lang=fr&zoom=&type=show_document",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/it/php/aza/http/index.php?highlight_docid=aza%3A%2F%2Faza://27-10-2021-8C_329-2021&lang=it&zoom=&type=show_document",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": "TF 8C_329/2021",
+				"creators": [],
+				"dateDecided": "27.10.2021",
```

Comment:
We should try to represent the dates as ISO 8601 dates as much as possible, so `YYYY-MM-DD`. Other date formats run into issues with internationalization when they're ambiguous. (On a US-locale machine, for instance, `11.10.2021` is parsed as November 10th, 2021`.)

Reply:
That was my oversight - corrected.